packages/runtime/container-runtime/src/gcSweepReadyUsageDetection.ts

```diff
@@ -17,28 +18,35 @@ import { oneDayMs } from "./garbageCollection";
  * Feature Gate Key -
  * How many days between closing the container from this error (avoids locking user out of their file altogether)
  */
-export const skipClosureForXDaysKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.SkipClosureForXDays";
+export const skipClosureForXDaysKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.SkipForXDays";
 
 /**
  * LocalStorage key (NOT via feature gate / monitoring context)
  * A map from docId to info about the last time we closed due to this error
  */
 export const closuresMapLocalStorageKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection.Closures";
 
+interface SweepReadyUsageDetectionConfig {
+    interactiveClient?: "close" | "crashOnLoadOnly";
+}
+
 /**
  * Feature gate key to enable closing the container if SweepReady objects are used.
  * Value should contain keywords "interactiveClient" and/or "summarizer" to enable detection in each container type
  */
-const sweepReadyUsageDetectionSetting = {
-    read(config: IConfigProvider) {
+ const sweepReadyUsageDetectionSetting = {
```

Comment:
nit: spacing

Reply:
I'm not confused, sad actually. But there is no "sad" reaction.

Reply:
It was probably the auto correct functionality on your code editor